packages/common/driver-definitions/package.json

```diff
@@ -63,35 +63,8 @@
 	},
 	"typeValidation": {
 		"broken": {
-			"TypeAliasDeclaration_DriverError": {
-				"backCompat": false
-			},
-			"EnumDeclaration_DriverErrorType": {
```

Comment:
Change kind 3: Type tests that now pass due to the more permissive comparison of string-like and number-like types.  The motivation is that TypeScript considers identical string/number literals as being unassignable to each other when coming from different (but identical) enum declarations, creating more noise than value in our backCompat tests.

In this case, the remaining forwardCompat breaking change remains, due to adding new values to the DriverErrorTypes enum-like, but types that are descriminated by 'DriverErrorTypes.someConstant' are no longer flagged (and shouldn't be, as the string value of 'someConstant' is unchanged.)